devicetypes/smartenit/smartenit-zigbee-metering-outlet.src/smartenit-zigbee-metering-outlet.groovy

```diff
@@ -0,0 +1,138 @@
+/*
+ *	Copyright 2016 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *	use this file except in compliance with the License. You may obtain a copy
+ *	of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software
+ *	distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *	WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *	License for the specific language governing permissions and limitations
+ *	under the License.
+ */
+ import groovy.transform.Field
+ 
+ @Field final MeteringCurrentSummation = 0x0000
+ @Field final MeteringInstantDemand = 0x0400
+ @Field final Current = 0x0508
+ @Field final Voltage = 0x0505
+ @Field final EnergyDivisor = 100000
+ @Field final CurrentDivisor = 1000
+ @Field final SmartenitMfrCode = 0x1075
+ @Field final ElectricalMeasurement = 0x0b04
+ @Field final ACVoltageMultiplier = 0x0600
+ @Field final ACVoltageDivisor = 0x0601
+ @Field final ACCurrentMultiplier = 0x0602
+ @Field final ACCurrentDivisor = 0x0603
```

Comment:
Those constants are not used in the code. Did You miss to support those values ? If not, please remove them.